src/optimizer/cost_and_stats_calculator.cpp

```diff
@@ -32,9 +32,21 @@ void CostAndStatsCalculator::CalculateCostAndStats(
   gexpr->Op().Accept(this);
 }
 
-void CostAndStatsCalculator::Visit(const PhysicalScan *) {
+void CostAndStatsCalculator::Visit(const DummyScan *) {
   // TODO: Replace with more accurate cost
   output_stats_.reset(new Stats(nullptr));
+  output_cost_ = 0;
+}
+
+void CostAndStatsCalculator::Visit(const PhysicalSeqScan *) {
+  // TODO: Replace with more accurate cost
+  output_stats_.reset(new Stats(nullptr));
+  output_cost_ = 2;
+};
+void CostAndStatsCalculator::Visit(const PhysicalIndexScan *) {
+  // TODO: Simple cost function
+  // indexSearchable ? SeqScan : IndexScan
```

Comment:
Yes. I think we should check whether the query is really index searchable or not here. If not, then the cost of this operator should be higher than SeqScan, instead of "1". Otherwise we'll choose the wrong plan even with simple benchmarking. Is it hard to add that logic into this PR?